### DISCUSSION TITLE
Too much blind/confusion spam

### DISCUSSION THREAD

**Discussion by lunasophia** [angband/angband#5693] [General]
I can't make a post on the forums (or even search the forums to see if this has been discussed before) because I can't register on the forums because registration is closed, so... here I am.

I usually play 3.5.1 but I wanted to give a new version a try. I started a half-troll warrior and got to about D30 or so. It was interesting to see all the changes from 3.5. Some stuff is really cool! The dungeon levels feel more interesting, monsters are a lot more interesting, they have more abilities.

... too many, really. I found myself *constantly* confused and blinded. I'd cure the status effect and then the very same turn it'd be reapplied. This happened again and again where I'd spend turn after turn just drinking a potion and hoping I'd be able to act this turn. I didn't find any items that protect against these statuses. All I could do was spam cure light wounds potions. Not for the healing--I generally stayed at 90+% HP--but just to cure the status effects. I can't really function when blind and especially not when confused, and seeing it spammed so much was so annoying that I actually quit and uninstalled the game. The game isn't fun like this.

Can we please consider rebalancing these status effects to be less prevalent?

**Comment by NickMcConnell** [comment]
On the forum issue - there are steps afoot to get it fixed, but it's slow.  On the blind/confuse issue, I'm sorry but that's the new way that it is.  I'd recommend playing different race/class combinations, like Dwarf for the innate RBlind.
